release_notes.md

```diff
@@ -19,4 +19,6 @@
   - `Microsoft.Azure.WebJobs` updated to 3.0.41
   - `Microsoft.Azure.WebJobs.Host.Storage` updated to 5.0.1
   - `Microsoft.Extensions.Azure` updated to 1.7.1
-  - `Azure.Storage.Blobs` updated to 12.19.1
\ No newline at end of file
+  - `Azure.Storage.Blobs` updated to 12.19.1
+- [in-proc] Updating FunctionsNetHost (dotnet isolated worker) to 1.0.9 (#10263)
+- Fix branch naming check for suffix (#10320)
```

Comment:
this doesn't need to be part of release notes

Reply:
I have removed the commit notes for fixing the branch as part of release notes